test/JDBC/expected/db_collation/test_like_index-vu-verify.out

```diff
@@ -0,0 +1,972 @@
+
```

Comment:
1. Should ensure to add test cases for toast rows. Verify that it picks up index scan or bitmap index scan for Toasted entries.
2. Have complex index on multiple columns and check that query picks up index scan if predicate with combination from those column is there 
3. Have a join between two tables with LIKE in join predicate. Verify that inner side pick up index scan for outer join. Or even for hash join

Reply:
Added in latest rev except 1st one (toastable entry) as it is not always possible to make sure that there are entries in toast table